services/blockchain-connector/shared/sdk/events.js

```diff
@@ -18,16 +18,30 @@ const { Signals } = require('lisk-service-framework');
 const { getRegisteredEvents } = require('./endpoints');
 const { getApiClient } = require('./client');
 
+const EVENT_CHAIN_FORK = 'chain_forked';
+const EVENT_CHAIN_BLOCK_NEW = 'chain_newBlock';
+const EVENT_CHAIN_BLOCK_DELETE = 'chain_deleteBlock';
+const EVENT_CHAIN_VALIDATORS_CHANGE = 'chain_validatorsChanged';
+const EVENT_TX_POOL_TRANSACTION_NEW = 'txpool_newTransaction';
+
+const events = [
+	EVENT_CHAIN_FORK,
+	EVENT_CHAIN_BLOCK_NEW,
+	EVENT_CHAIN_BLOCK_DELETE,
+	EVENT_CHAIN_VALIDATORS_CHANGE,
+	EVENT_TX_POOL_TRANSACTION_NEW,
+];
+
 const subscribeToAllRegisteredEvents = async () => {
 	const apiClient = await getApiClient();
 	const registeredEvents = await getRegisteredEvents();
-
-	registeredEvents.forEach(event => {
+	const allEvents = registeredEvents.concat(events);
+	allEvents.forEach(event => {
 		apiClient.subscribe(
 			event,
 			payload => Signals.get(event).dispatch(payload),
 		);
 	});
 };
 
-module.exports = { subscribeToAllRegisteredEvents };
+module.exports = { subscribeToAllRegisteredEvents, events };
```

Comment:
Do we need to explicitly export `events`? `subscribeToAllRegisteredEvents` seems to concatenate them while processing the remaining.

Reply:
Added `TODO` to update the logic once generic implementation for events is ready and updated the issue description as well!